package.json

```
@@ -23,9 +23,10 @@
     "test": "jest",
     "pg:up": "docker compose up -d",
     "pg:down": "docker compose down",
-    "pg:makemigrations": "npx prisma migrate dev --name $1",
-    "pg:migrate": "npx prisma migrate deploy",
-    "pg:client:generate": "npx prisma generate",
+    "typeorm": "typeorm-ts-node-commonjs",
+    "migration:generate": "npx typeorm-ts-node-commonjs migration:generate -d ./data-source.ts",
```

npx는 `Node Package eXecuter`의 약자인데, cli같이 일회성 실행이 필요한 경우 사용하는 명령어야. 위에서 형이 `typeorm` 이라는 패키지 스크립트를 만들었기 때문에 alias로  `npx typeorm-ts-node-commonjs` 를 `npm run typeorm ~~` 과 같이 짧게 만들어줄 수 있어

---

이해했어용 굿 수정 하겠슴다

---

@J-Hoplin 이거 좀 헷갈리는데, typeorm을 미리 선언해둬서 아래 migrate나 run, revert에서 재사용 가능한건 이해가 됐는데,
그럼 아래 두개중에 어떤게 맞는거야? 둘다 맞나?
```suggestion
    ### 1번
    "migration:generate": "npm run typeorm migration:generate -d ./data-source.ts",
    ### 2번
    "migration:generate": "migration:generate -d ./data-source.ts -d ./data-source.ts",
```
1번은 migration:generate만 쓰면 되고, 2번은 npm run typeorm migration:generate 를 쓰면 되는걸까?

---

엉엉 형이 지금 한대로가 맞아 npx가 커맨드를 1회성 실행하는거는 우리가 package.json에 정의한 스크립트 커맨드에 대한게 아닌 외부 cli package들에 대해서야 예를들어 nest-cli도 이중 하나고